cmd/contour/serve.go

```diff
@@ -251,6 +251,11 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	// register observer for endpoints updates.
 	endpointHandler.Observer = contour.ComposeObservers(snapshotHandler)
 
+	dnsLookupFamily, err := ParseDNSLookupFamily(ctx.DNSLookupFamily)
+	if err != nil {
+		log.WithField("context", "ConfigurationFile").WithField("name", "cluster.dns-lookup-family").Errorf("error parsing parameter dns-lookup-family: %s", err)
```

Comment:
I'm still thinking that returning the error rather than just logging it is more consistent with how we handle parse errors on other config file fields - failures to parse cause `contour serve` to exit. Above all I just prefer consistency.

Reply:
It now returns an error instead of logging.